h2o-docs/src/product/data-science/gbm-faq/reproducibility.rst

```diff
@@ -64,3 +62,33 @@ Reproducibility
    
    - Refer to :ref:`save_and_load_model` for more information.
    - The binary model will contain the H2O version and the parameters used to train the model.
+
+ 4. Train the GBM model using the same:
+	
+   - training data (information about the training data saved in Step 1)
+   - cluster configuration (information about the cluster configuration saved in Step 2)
+   - same parameters (information about the parameters saved in Step 3)
```

Comment:
I think you can remove "same" from this bullet:
> same parameters (information about the parameters saved in Step 3)

Reply:
thanks for the catch! Just fixed it